crates/components/deskc-ast/Cargo.toml

```diff
@@ -14,3 +14,4 @@ edition = "2021"
 ids = { path = "../deskc-ids", version = "0.0.0", package = "deskc-ids" }
 dson = { path = "../dson", version = "0.0.0", package = "dson" }
 thiserror = "1.0.37"
+parol_runtime = { git = "https://github.com/jsinger67/parol", rev = "e0c4a1d59bd6dc05b5ba9f85d28e1e1bbdf78d5a", features = ["auto_generation"] }
```

Comment:
In this project, components should depend on crates as little as possible for cleanliness and faster compile time. We need a way around this dependency.